### ISSUE TITLE
Sensors reporting too often

### ISSUE THREAD

**Issue by omniwired** [Koenkk/zigbee2mqtt#684]
The sensors with problem is the Aqara Humidity sensor.
It basically is the opposite problem of #103 

Is being 3 days since the first install and is eating the battery very (very) quickly from 100 to 37 in 3 days.

Is there a way to make them report less often ?

These sensors never connected to any other condinator other than the zigbee2mqtt stick and firmware.

Stack:
- Home assistant
- Intel Nuc
- zigbee2mqtt using docker
- mosquitto
- 6 devices on the network, no ZigBee routers.

Example output:
> {"temperature":17.34,"linkquality":76,"humidity":61.29,"pressure":1010,"battery":37,"voltage":2975}
{"temperature":17.34,"linkquality":73,"humidity":61.65,"pressure":1010,"battery":37,"voltage":2975}
{"temperature":17.34,"linkquality":73,"humidity":61.65,"pressure":1010,"battery":37,"voltage":2975}

**Comment by ryanbeaton** [comment]
I think you will find they are not reporting too often it is just the calculation of the battery percentage is out. There has been some debate here about how to calculate. 3000 for voltatage is 100% so you’re ok.

**Comment by omniwired** [comment]
I see; the battery started at 3025 tho.

**Comment by Koenkk** [comment]
As @ryanbeaton mentions, the battery calculation is indeed off. 

@omniwired please monitor this for a few days and check if the voltage drops further.

**Comment by omniwired** [comment]
After a couple days:
> zigbee2mqtt/living {"temperature":18.01,"linkquality":49,"humidity":39.34,"pressure":1016,"battery":31,"voltage":2965}

**Comment by Koenkk** [comment]
@omniwired how many times do they report?

**Comment by cynicated** [comment]
I've noticed that my Aqara sensors tend to report anytime there's a significant change in one of their monitored conditions. I have the temperature, humidity & pressure sensor (3 in 1) and it reports whenever any of those change. 

In fact, I ended up moving the sensor in my mud room because it's unheated and sealed off from the rest of the house, but also very small. So whenever we open the door into the house the temperature fluctuates enough for the sensor to send a lot of readings while it raises, and a lot while it lowers. Which gives me a bunch of weird spikes in the graph (these things seem very sensitive). It just wasn't the type of information I really was looking for...
